### ISSUE TITLE
Ban wave coming

### ISSUE THREAD

**Issue by avexus** [PokemonGoF/PokemonGo-Bot#5746]
https://www.google.com/amp/s/b.marfeel.com/www.i4u.com/2016/09/115802/pokemon-go-ban-wave-underway-hitting-bots%3fmarfeeltn=amp?client=ms-android-samsung

**Comment by aaronluna18** [comment]
yep i have 2 ban accounts now

**Comment by avexus** [comment]
Sorry to hear that

**Comment by avexus** [comment]
I would say it's safe to not bot for a while. They have a way to flag which accounts uses bot / sniper. It's for them to decide when to ban.

**Comment by marcotini** [comment]
Do you have random pause active?

**Comment by avexus** [comment]
Why?

**Comment by MerlionRock** [comment]
Your 2 accounts that got banned, they snipe?

**Comment by Paradoxum** [comment]
First/only account, level 26, have sniped allover the world, still able to login - origin UK.

**Comment by avexus** [comment]
@marcotini why does random pause matter?

**Comment by Fanlabiz** [comment]
Do they currently detect that we play with an IP address which is not from the country where the bot is playing ?

**Comment by iruy** [comment]
@Fanlabiz I'd say that almost for sure!!

**Comment by aaronluna18** [comment]
@MerlionRock YES in both I used the snipe! but I have 3 accounts working fine just one of that 3 I used snipe.

**Comment by marcotini** [comment]
@avexus it's matter because they can not ban somethings that do things at random. If you abuse it's obvious that ban is coming.
Try to make more real than possibile. If you have a wrong config file, the ban is on their way.

**Comment by Paradoxum** [comment]
anyone from the UK had a bot banned?

**Comment by camnomis** [comment]
I am UK based and use the bot, will hold off for the time being (well might setup a brand new account to see if it gets banned), I don't know if it helps I mix up logging on through the app and though the bot which is keeping me off Niantic radar. Its a shame as starting to use the bot more of an "admin" tool rather than levelling, I still like the excitement of actually playing the game and catching things, but use the bot to sort and transfer the lower grade ones and auto evolve the higher spec based on rules etc. 

I think of it more as a mod (used to playing WOW etc where mods are accepted and even encouraged as it enhances game play) than a cheat, I had a "bot" account that I levelled up very quickly but had no affinity to the Pokemon, where as I know where I was when my Snorlax hatched, or where I caught the last Magikarp needed to evolve my Gyarados

I guess by using the simple act of using API that causes the risk of the ban, not the actions that are under taken

**Comment by Paradoxum** [comment]
> Its a shame as starting to use the bot more of an "admin" tool rather than levelling,

I know what you mean. I have stopped actually botting with it and just using it to check the IV/CP/IVCP of my pokemon without having to run each one through a calculator individually etc..

**Comment by darthmamour** [comment]
just reading

**Comment by ProZer0** [comment]
@PokemonGoZ I wasn't aware that you need to crack Unknown6 , I can't find anywhere on the web that there is an issue with the current pgoapi-1.1.6 . Could you please give us more details?

Thanks!

**Comment by bsjpark** [comment]
How do you check if your repo is on v0.39 API or v0.35 API?

**Comment by ShadyBS** [comment]
No repos are in 0.39  because the API is not cracked as of yet. @bsjpark

**Comment by MerlionRock** [comment]
It's not cracked since 0.37 API
By the time they crack it, Niantic will most likely come out something different again... Not looking good for botting community...

**Comment by bsjpark** [comment]
Sounds like safe thing to do is not to bot. What a sad day.

**Comment by avexus** [comment]
3 of my active bot accounts used this bot and 0.35 api. Even botted for 3 hours today. They all survived, for now. Will see what happens. I think they will catch me.

**Comment by MerlionRock** [comment]
@avexus I think it's a trap. To create enough buffer time to distingue who's using their latest app and who's not.

**Comment by avexus** [comment]
@MerlionRock I think so too. That's why ban comes in waves. They accumulate enough data. Then, turn on one parameter in their conf file {set ban="true"}, then run their bot.

**Comment by iruy** [comment]
So, they are botting too :)

**Comment by ravey-davey** [comment]
Surely older versions of the actual game will still run the older API (in the case where people have not updated it). It would seem a little odd to ban users on the basis of the API being used to interface into PG on that basis alone..?

**Comment by MerlionRock** [comment]
@ravey-davey they are forcing all to use latest version. You can't play with older version of the app.

**Comment by ProZer0** [comment]
Botters in Sweden, hold your breath! The ban has just started, two bots of mine were banned before few minutes o_O

**Comment by tarakanUnlimited** [comment]
Got myself new account just to test bans. I saw that nobody spawned and all stops were out of range. Logging in from phone asked me to solve the captcha.

**Comment by sullydog101** [comment]
Just today my level 28 account wasn't working on the bot (would log in but couldn't get any data) I logged in on my phone and got a crazy notification ,"We have detected signals from your account that suggests you are using third party applications or modified software typically associated with cheating. Please be aware..." (It's longer than that but that is thegist of it) had been botting locally for about six hours today.

**Comment by sullydog101** [comment]
Made me do a captcha too

**Comment by sullydog101** [comment]
Wow they hard flagged my account... The same message pops up every time I log in with it now, but not on my other account that I haven't botted with in a few days. Still, the message is better than being permabanned

**Comment by MerlionRock** [comment]
@sullydog101, at least you have a msg...

Mine just couldn't login at all...

**Comment by pacpac42** [comment]
@sullydog101 can you post the full notification you got? is this hard flag a temporary ban?

**Comment by dogeatdog** [comment]
[https://www.dropbox.com/s/lu3q6lwqb3sccky/Schermafdruk%202016-10-06%2007.33.06.png?dl=0](url)

Same here. Got the captcha once, but the message pops up every time now.

**Comment by avexus** [comment]
@iruy， they cannot do the banning manually.

**Comment by mmixx** [comment]
Yeah... I haven't used the bot with my 2 accounts for a few days until now and I haven't received a warning or even a captcha prompt...

**Comment by sullydog101** [comment]
Dogeatdog beat me to it, same message. Pops up every time on that account when you first log in. After you click the button though you can play completely normally.
